pkg/types/powervs/machinepools.go

```diff
@@ -0,0 +1,51 @@
+package powervs
+
+// MachinePool stores the configuration for a machine pool installed on IBM Power VS.
+type MachinePool struct {
+	// VolumeIDs is the list of volumes attached to the instance.
+	//
+	// +optional
+	VolumeIDs []string `json:"volumeIDs"`
+
+	// Memory defines the memory in GB for the instance.
+	//
+	// +optional
+	Memory string `json:"memory"`
```

Comment:
Why is this a string instead of an int64?

Reply:
i can't remember specifically what, but something somewhere complained and i changed it months ago.

Reply:
Please either detail why this needs to be a string or make it an int64.